client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -12,24 +12,27 @@ export const CommentDetailActions = ( {
 	commentIsLiked,
 	commentIsSpam,
 	commentIsTrash,
+	deleteForever,
 	toggleApprove,
 	toggleLike,
 	toggleSpam,
 	toggleTrash,
 	translate,
 } ) =>
 	<div className="comment-detail__actions">
-		<a
-			className={ classNames( 'comment-detail__action-like', { 'is-liked': commentIsLiked } ) }
-			onClick={ toggleLike }
-		>
-			<Gridicon icon={ commentIsLiked ? 'star' : 'star-outline' } />
-			<span>{
-				commentIsLiked
-					? translate( 'Liked' )
-					: translate( 'Like' )
-			}</span>
-		</a>
+		{ ! commentIsSpam && ! commentIsTrash &&
```

Comment:
How would folks feel about passing through an explicit list of actions to render? I think we could avoid having to pass through so many booleans then, if we pass through status, isLiked, and an action list.

Where actions might look like:
```
 [ 'like', 'approve', 'spam' 'trash' ]
```
or
```
 [ 'approve', 'delete' ]
```